server/fishtest/templates/actions.mak

```diff
@@ -1,4 +1,4 @@
-<%inherit file="base.mak"/>
+-<%inherit file="base.mak"/>
```

Comment:
I wonder about this change.

Reply:
Confirmed typo

![image](https://user-images.githubusercontent.com/15718418/210168737-71954e73-5f8f-4a40-9a15-a9b3868902ea.png)

Reply:
Fixed!